programs/bpf_loader/src/syscalls.rs

```diff
@@ -921,6 +923,9 @@ impl<'a> SyscallInvokeSigned<'a> for SyscallInvokeSignedRust<'a> {
                 signers_seeds_len,
                 self.loader_id,
             )?;
+            if signers_seeds.len() > MAX_SEEDS {
```

Comment:
This isn't the right check. This is limiting the number of signers rather than the number of seeds for a given signer. We need a check below for `untranslated_seeds.len() > MAX_SEEDS`.

But we should also limit the number of signers.. probably to the the length of keyed accounts?